app/models/mixins/scanning_mixin.rb

```diff
@@ -119,7 +119,7 @@ def scan_queue(userid = "system", options = {})
     )
   end
 
-  # Call the miqhost webservice to do the SyncMetadata operation
+  # Do the SyncMetadata operation through the server smart proxy
```

Comment:
LOL, yeah, I don't know if we need these comments but they were lies so I fixed them... I saw them when I git grepped form the SyncMetadata constant found in the Verbs directory.